integration/bundle/bind_resource_test.go

```diff
@@ -17,8 +17,70 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/databricks-sdk-go/service/ml"
 )
 
+func TestBindExperimentToExistingExperiment(t *testing.T) {
```

Comment:
I know that existing tests for bind are integration tests but have you explored making new ones acceptance tests instead?